database/L/Little Simz/GREY Area/Venom

```diff
@@ -0,0 +1,91 @@
+Life sucks and I never tried suicide
+Minds fucked even more than I realise
+Times up keep it moving when she arrives
+If you ever heard what I heard in my mind
+Never try
+You would cry
+Thats a lie 
+You would die
+I don’t ever wanna come down from a high
+Im in the best seat
+From time been a next breed
+If your coming come at me directly
+Don’t need no-one to defend me 
+Souls in a place even I can’t get to
+Don’t fuck with the deadly
+Moon waves and open seas
+Quick coasting over seas
+Fuck those who don’t believe
+They will never wanna admit I’m the best here
+For the mere fact that I’ve got ovaries
+Its a woman world so to speak
+Pussy you sour 
+Never giving credit where its due cos you don’t like pussy in power
+Venom 
+Venom
+
+My my if you ever heard what I heard in the night
+What a fright
+Must’ve been a parasite in my past life
+I don’t ever wanna come down from a high
+My soul ascended to the sky
+Its just you and I
+Gone but I’m feeling too alive
+Tryna get me outta spite
+Someones gotta pay I ain’t talking big amounts
+Some kinda physical pain
+Some kinda traumatic shit
+Niggas wanna see dead bodies probably not they ain’t rolling with no automatic clips
+Moving scatty all erratic and shit
+Niggas pussy licking batty and shit
+Oh you mad? Then come at me you prick
+Make a move better pattern it quick
+I assume you’ll be coming for blood 
+That makes 2 of us that makes 2 of us
+Venom 
+
+Venom
+Venom
+Venom 
+
+VERSE 2
```

Comment:
This is not part of the text, if it's not audible in the song it perhaps should be removed